lhotse/recipes/voxceleb.py

```diff
@@ -0,0 +1,333 @@
+"""
```

Comment:
Is it possible to add a download voxceb function?

Reply:
I have added it now. It worked a few times (I didn't run it through to completion though), but then I started getting "connection refused" errors, so it seems the server is not always reliable (or requires some password). I will add this warning in the function dosctring.